gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Tue Nov 14 15:33:28 CET 2017
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
```

Comment:
Why not use the latest version of gradle?

Reply:
133c2410b2b59c03a309b086ea3396bfacf026f8